en/components/general/getting_started.md

```diff
@@ -0,0 +1,230 @@
+
+### Add components automatically
+
+#### Import modules and use components
+In order to easily add new components to our application, we can take advantage of the Ignite UI CLI!
+
+```cmd
+ig add
+```
+
+> [!NOTE]
+> Please note that the [`ig add`](https://github.com/IgniteUI/igniteui-cli/wiki/add) command is supported only on an existing project created with Ignite UI CLI.
+
+After going through the options of the menu and choosing which component we want to add to our application, we will notice that we have a brand new component in our project, which we can use anywhere on our page!
+
+#### Run application
+
+Now let’s run our application with the [`ig start`](https://github.com/IgniteUI/igniteui-cli/wiki/start) command to see our awesome page!
+
+```cmd
+ig start
+```
+
+### Add components manually
+
+#### Import modules
+
+First we have to import the respective modules of the components we want to use in the **app.module.ts** file. We will go ahead and do this for the [**igxGrid**](../grid.md)! Please note that some components have animations depending on BrowserAnimationsModule, so let’s import that one as well:
+
+```typescript
+// app.module.ts
+
+import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { NgModule } from '@angular/core';
+import { AppComponent } from './app.component';
+
+// Here we import the IgxGridModule, so we can use the igxGrid!
+import { IgxGridModule } from 'igniteui-angular';
+
+@NgModule({
+  declarations: [
+    AppComponent
+  ],
+  imports: [
+    BrowserModule,
+    BrowserAnimationsModule,
+    IgxGridModule
+  ],
+  providers: [],
+  bootstrap: [AppComponent]
+})
+export class AppModule { }
+```
+
+#### Use components
+
+We are now ready to use the igxGrid in our markup! Let's go ahead and define it in our **app.component.html** file:
+
+```html
+<!-- app.component.html -->
+
+<div style="text-align:center; margin-bottom: 20px;">
+  <h1>
+    Welcome to {{title}}!
+  </h1>
+</div>
+
+<div style="text-align: center;">
+  <igx-grid [data]="localData" width="600px" height="400px" style="margin: auto" [allowFiltering]="true">
+    <igx-column field="Name" dataType="string"></igx-column>
+    <igx-column field="Age" dataType="number"></igx-column>
+  </igx-grid>
+</div>
+```
+
+We will also define the data of the grid and the title of our application that are referenced from the **app.component.ts**:
+
+```typescript
+// app.component.ts
+
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.css']
+})
+export class AppComponent {
+  localData = [
+    { Name:'John', Age: 29 },
+    { Name:'Alice', Age: 27 },
+    { Name:'Jessica', Age: 31 },
+  ];
+
+  title = 'My Ignite UI project';
+}
+```
+
+#### Run application
+
+Finally, we can run our new application by using the command below:
```

Comment:
Here we have two possible options. In case the developer has created a project using angular cli use ng serve, and if the project has been created using the igniteui-cli and for whatever reason the developer decided to not use the cli and add components manually he/she still can use the igniteui-cli to run ig start command to run the project.